spec/factories/git_repositories.rb

```diff
@@ -1,5 +1,8 @@
+require 'faker'
```

Comment:
I don't really like this `require` here, but I'm not sure where else to put it.  Anyone have suggestions? cc @carbonin @jrafanie 

Once this is merged I'd like to follow up with removing `ip_from_seq` and friends

Reply:
see 👇

Reply:
My only concern is it seems to load all of the gem in this require.  I don't know how costly this is:

https://github.com/stympy/faker/blob/f03d4eec1b11b8a03413c89b7d2cadcd0cfd6ef9/lib/faker.rb#L12
https://github.com/stympy/faker/blob/f03d4eec1b11b8a03413c89b7d2cadcd0cfd6ef9/lib/faker.rb#L253


I don't know if you can just require faker/internet

Reply:
Nope, I need to require the whole thing

Reply:
I tried locally and the memory growth is less than 2 MB.

It  requires 113 files but they appear to be small.

```
irb(main):002:0> $LOADED_FEATURES.length
=> 3337
irb(main):003:0> require 'faker'
=> true
irb(main):004:0> $LOADED_FEATURES.length
=> 3450
```